Add admin CSS styling for quote comments

pfefferle opened the pull request:
## Proposed changes:
* Add `.quote` selector to admin CSS rules for consistent comment styling
* Ensure quote comments display with the same indentation and spacing as likes and reposts
* Maintain alphabetical ordering of selectors (like, quote, repost)

| Before | After |
|---|---|
| <img width="614" height="125" alt="Screenshot 2025-12-04 at 09 25 02" src="https://github.com/user-attachments/assets/7a324157-ed97-448b-a2c7-6d8e8d912555" /> | <img width="605" height="90" alt="Screenshot 2025-12-04 at 09 25 12" src="https://github.com/user-attachments/assets/d37e7fbd-c001-4f88-b6d7-3343a9b26b71" /> |


### Other information:

- [ ] Have you written new tests for your changes, if applicable?

N/A - CSS styling change

## Testing instructions:

1. Create or receive a quote comment on a post (a quote is when someone quotes your post in their own post)
2. Go to **Comments** in WordPress admin
3. Locate a quote comment in the list
4. Verify that quote comments have:
   - Proper left padding (63px) matching likes and reposts
   - Proper comment author margin matching other interaction types
5. Compare the visual styling with likes and reposts to ensure consistency

**Before:** Quote comments may have had inconsistent spacing/indentation compared to likes and reposts

**After:** Quote comments display with identical spacing and indentation as other interaction types

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Added - for new features
-   [ ] Changed - for changes in existing functionality
-   [ ] Deprecated - for soon-to-be removed features
-   [ ] Removed - for now removed features
-   [x] Fixed - for any bug fixes
-   [ ] Security - in case of vulnerabilities

#### Message

Fix admin styling for quote comments to match likes and reposts

</details>

pfefferle committed:
Add admin CSS styling for quote comments

Extend the existing CSS rules for likes and reposts to also apply to quote comments, ensuring consistent styling across all interaction types in the WordPress admin.

Changes:
- Add .quote selector to dashboard-comment-wrap padding rule
- Add .quote selector to comment-author margin rule
- Maintain alphabetical ordering of selectors (like, quote, repost)

This ensures quote comments display with the same indentation and spacing as likes and reposts in the admin interface.

matticbot committed:
Add changelog

pfefferle committed:
Merge branch 'trunk' into fix/quote-comment-styling

obenland commented on assets/css/activitypub-admin.css:
```diff
@@ -230,13 +230,15 @@ input.blog-user-identifier {
-.like .dashboard-comment-wrap {
+.like .dashboard-comment-wrap,
+.quote .dashboard-comment-wrap,
+.repost .dashboard-comment-wrap {
```
```suggestion
.activitypub-comment .dashboard-comment-wrap {
```
We could also do that and be more forward compatible.

pfefferle committed:
Refactor activity comment CSS selectors

Unified the CSS selectors for .like, .quote, and .repost comment styles under .activitypub-comment to simplify styling and maintenance.

pfefferle committed:
Merge branch 'trunk' into fix/quote-comment-styling
